src/db/clients/mysql.js

```diff
@@ -127,6 +128,22 @@ export async function listTableTriggers(conn, table) {
   return data.map((row) => row.trigger_name);
 }
 
+export async function listTableIndexes(conn, table) {
+  const schema = await getSchema(conn);
+
+  const sql = `
+  SHOW INDEX FROM ${table} FROM ${schema};
```

Comment:
I had some problems here, when i use `?` the execution goes like that
`SHOW INDEX FROM 'table_name' FROM 'database_name';`  an it does not work in mysql, mysql need this `SHOW INDEX FROM table_name FROM database_name;`  without quotes.
So i used the mysql.escape() that prepares the param and protect from injection, then removed the quotes. It now looks like that.
`SHOW INDEX FROM ${mysql.escape(table).replace(/'/g, '')} FROM ${mysql.escape(database).replace(/'/g, '')} ;`

What do you think?